extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -52,25 +53,32 @@ function PanelMeasurementTableTracking({ servicesManager, extensionManager }) {
     debouncedMeasurementChangeTimestamp,
   ]);
 
-  // ~~ DisplayStudySummary
-  useEffect(() => {
+  const test = async () => {
```

Comment:
test?

Reply:
So, testing to see if you were paying attention 🥇  👀